pkg/oc/clusterup/run_self_hosted.go

```diff
@@ -68,8 +69,27 @@ var (
 		},
 	}
 
+	runlevelZeroLabel         = map[string]string{"openshift.io/run-level": "0"}
 	runlevelOneLabel          = map[string]string{"openshift.io/run-level": "1"}
 	runLevelOneKubeComponents = []componentInstallTemplate{
+		{
+			ComponentImage: "hypershift",
```

Comment:
this will be removed in bootkube cluster up... the etcd is started manually in container :-)